_index.md

```
@@ -136,7 +136,11 @@ summary: TiDB is an open-source distributed SQL database that supports Hybrid Tr
 
 [System Variables](https://docs.pingcap.com/tidb/dev/system-variables)
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.23.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)